src/utils/analytics.ts

```diff
@@ -106,56 +114,52 @@ export function recordWalletConnect(accountId: string) {
   }
 }
 
-export function reset() {
-  if (!segment) return;
+export function logout() {
+  if (!rudderAnalytics) return;
   try {
     recordEvent('wallet-logout');
     localStorage.removeItem('hashId');
-    localStorage.removeItem('anonymousUserId');
-    localStorage.removeItem('anonymousUserIdCreatedAt');
-    segment.flush();
+    rudderAnalytics.reset();
   } catch (e) {
     console.error(e);
   }
 }
 
-export function flushEvents() {
-  if (!segment) return;
-  return segment.flush();
+export function reset() {
+  if (!rudderAnalytics) return;
+  return rudderAnalytics.reset();
 }
 
-function recordEventWithProps(eventLabel: string, properties: Record<string, unknown>) {
-  if (!segment) return;
+function recordEventWithProps(eventLabel: string, properties: Record<string, string>) {
+  if (!rudderAnalytics) return;
   try {
-    segment.track({
-      anonymousId: getAnonymousId(),
-      event: eventLabel,
-      properties: {
+    rudderAnalytics.track(
+      eventLabel,
+      {
         ...properties,
         hashId: localStorage.getItem('hashId'),
-        anonymousUserIdCreatedAt,
-      },
-    });
+        anonymousUserIdCreatedAt
+      }
+    );
   } catch (e) {
     console.error(e);
   }
 }
 
 export function recordEvent(eventLabel: string) {
-  if (!segment) return;
+  if (!rudderAnalytics) return;
   try {
-    segment.track({
-      anonymousId: getAnonymousId(),
-      event: eventLabel,
-      properties: {
+    rudderAnalytics.track(
+      eventLabel,
+      {
         hashId: localStorage.getItem('hashId'),
         url: window.location.href,
         anonymousUserIdCreatedAt,
-      },
-    });
-  } catch (e) {
-    console.error(e);
-  }
+      }
+    );
+} catch (e) {
```

Comment:
this formatting doesn't look right. Please make sure you have `prettier` installed in your editor with it automatically running on save. Once that is enabled, please open each file touched in this PR and save it then commit any resulting changes